tests/test_preprints.py

```diff
@@ -25,31 +27,52 @@ class TestPreprintWorkflow:
 
     @markers.dont_run_on_prod
     @markers.core_functionality
-    @pytest.mark.skip('Update/run once NPD is finalized.')
     def test_create_preprint_from_landing(self, driver, landing_page, project_with_file):
+
         landing_page.add_preprint_button.click()
         submit_page = PreprintSubmitPage(driver, verify=True)
+
+        # Wait for select a service to show
+        WebDriverWait(driver, 10).until(EC.visibility_of(submit_page.select_a_service_help_text))
         submit_page.select_a_service_save_button.click()
         submit_page.upload_from_existing_project_button.click()
         submit_page.upload_project_selector.click()
         submit_page.upload_project_help_text.here_then_gone()
         submit_page.upload_project_selector_project.click()
-        submit_page.upload_existing_file_button.click()
+
         submit_page.upload_select_file.click()
-        submit_page.convert_existing_component_button.click()
-        submit_page.continue_with_this_project_button.click()
-        submit_page.upload_save_button.click()
+        submit_page.upload_file_save_continue.click()
+
+        submit_page.basics_license_dropdown.click()
+        submit_page.basics_universal_license.click()
+        submit_page.basics_tags_section.click()
+        submit_page.basics_tags_input.send_keys('selenium\r')
+        submit_page.basics_abstract_input.click()
+        submit_page.basics_abstract_input.send_keys('Center for Open Selenium')
+        submit_page.basics_save_button.click()
 
+        # Wait for discipline help text
```

Comment:
I can tell from this comment 
`# Wait for discipline help text`
and the others below it that the intention is to wait for the element before proceeding with the test steps, but the wait hasn't been coded yet